BUILDING.md

```diff
@@ -65,22 +72,29 @@ The build output will be in the directory you specified with `do config`.
 
 ### Install Android SDK
 * Unzip the [Android SDK](https://dl.google.com/android/repository/tools_r25.2.3-macosx.zip) to a directory of your choosing.
+  * Instead, if you have Homebrew, you can install via `brew install android-sdk` and the default installation location would be `$HOMEBREW_PREFIX/opt/android-sdk`.
 * Using `tools/android` download:
   * Android 5.0.1 (API 21)
     * SDK Platform
   * Android SDK Build-tools 21.1.2
 
 ### Install Android NDK
 * Unzip the [Android NDK](https://dl.google.com/android/repository/android-ndk-r13b-darwin-x86_64.zip) to a directory of your choosing.
+  * Instead, if you have Homebrew, you can install via `brew install android-ndk` and the default installation location would be `$HOMEBREW_PREFIX/opt/android-ndk`.
 
-### Install [CMake](https://cmake.org/files/v3.7/cmake-3.7.1-Darwin-x86_64.dmg).
+### Install [CMake](https://cmake.org/files/v3.7/cmake-3.7.1-Darwin-x86_64.dmg)
+  * Instead, if you have Homebrew, you can install via `brew install cmake`.
```

Comment:
Oh, really? That's weird. Do you know the reason? CMake 3.7.2 (from HomeBrew) seems working fine for me.

When I see that a project needs Softwrae x.y, I would expect Software > x.y should work and install a more recent one if exists, unless the project specifically says only x.y. And I assume it's not only me thinking like this. Should we be more specific in the instruction about the version then?